plugins/resin/src/main/java/com/navercorp/pinpoint/plugin/resin/interceptor/ServletInvocationInterceptor.java

```diff
@@ -106,23 +119,21 @@ public void before(Object target, Object[] args) {
 			}
 		}
 	}
-	
-	
-	public boolean checkRequestDispatcherImpl(){
+
+	public boolean checkRequestDispatcherImpl() {
```

Comment:
Retrieving stack traces every request will be a big hit in performance.
This method won't be needed after you add `ServletInvocationInterceptor` as a scoped interceptor (see the comments in `ResinPlugin` for details on this).